apps/explorer/src/pages/validators/Validators.tsx

```diff
@@ -81,7 +105,11 @@ function ValidatorPageResult(): JSX.Element {
     const lastEpochRewardOnAllValidators =
         epochData?.data[0].endOfEpochInfo?.totalStakeRewardsDistributed;
 
-    const tableData = data ? [...data.activeValidators].sort(() => 0.5 - Math.random()) : [];
+    if (data && Number(data.pendingActiveValidatorsSize) > 0) {
+        activeValidatorsData = [...data.activeValidators, ...sanitizePendingValidatorsData];
+    }
+
+    const tableData = data ? activeValidatorsData?.sort(() => 0.5 - Math.random()) : [];
```

Comment:
just comment, it would make sense to me to show the pending validators at the end of the list. wdyt?

![image](https://github.com/user-attachments/assets/d3075d51-ba64-47fb-b351-fca6469e4864)